cpp/include/cudf/io/csv.hpp

```diff
@@ -1332,7 +1332,7 @@ class csv_writer_options {
   size_type _rows_per_chunk = std::numeric_limits<size_type>::max();
   // character to use for separating lines (default "\n")
   std::string _line_terminator = "\n";
-  // character to use for separating lines (default "\n")
+  // character to use for separating column values (default ",")
```

Comment:
There's a copy-paste error in all _inter_column_delimiter functions in this file. Would you mind correcting them too, since you started here? :grin:

Reply:
Updated.